onnxscript/ir/_core.py

```diff
@@ -1520,8 +1518,11 @@ def const_value(
     @const_value.setter
     def const_value(
         self,
-        value: _protocols.TensorProtocol | Sequence[_protocols.TensorProtocol] | None,
```

Comment:
Is Sequence[...] not necessary?

Reply:
Using the sequence annotation made operations like value.const_value.name=… difficult because we would need to assert that const_value is not a sequence everytime. I am removing this for now and will consider better ways when we need const value for sequences